android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerRootHelper.kt

```diff
@@ -78,12 +78,14 @@ class RNGestureHandlerRootHelper(private val context: ReactContext, wrappedView:
         action = MotionEvent.ACTION_CANCEL
       }
       if (rootView is RootView) {
-        rootView.onChildStartedNativeGesture(event)
+        rootView.onChildStartedNativeGesture(this.view, event)
```

Comment:
I'm not quite sure what should be passed here. Deprecated method calls this one with `null`, maybe we could also do that?

Reply:
Ideally, I think we should pass a view of the activating gesture but I don't think we can access it here. I guess passing the root view is ok though, since we are certain that the view is its child.